main.go

```diff
@@ -33,9 +32,9 @@ const (
 )
 
 var (
-	maxParallelTasks = flag.Int("p", runtime.NumCPU(), "max number of tasks to run in parallel")
-	maxLogLines      = flag.Int("max-log-lines", defaultMaxLogLines, "max number of log lines fetched with oc logs")
-	printVersion     = flag.Bool("version", false, "print version and exit")
+	concurrentTasks = flag.Int("p", runtime.NumCPU(), "number of tasks to run concurrently")
```

Comment:
"parallel" was misleading. One can call with "-p 1000" and it doesn't mean that there will be parallel execution of 1000 tasks, unless there are 1000+ CPU cores.